canonical-web-and-design/charmed-kubeflow.io
# Add the missing prism and fix docs links

## Done
- Added prism back to the project
- Fixed the docs links

## QA
- Check that the changes will resolve the [reported docs missing links](https://github.com/canonical-web-and-design/charmed-kubeflow.io/runs/4395325450?check_suite_focus=true)
- Go to the docs in the demo and see there is no error loading the prism file.
- Other docs links are fixed via discourse.
